server/monitor.go

```diff
@@ -64,8 +67,32 @@ func (s *Server) HandleConnz(w http.ResponseWriter, r *http.Request) {
 	s.mu.Lock()
 	c.NumConns = len(s.clients)
 
+	// Copy the keys to sort by them
+	pairs := make([]Pair, c.NumConns)
 	i := 0
-	for _, client := range s.clients {
+	for k, v := range s.clients {
+		pairs[i] = Pair{Key: k, Val: v}
```

Comment:
Ok, but would have to remove initializing to length `make([]Pair, c.NumConns)` since pairs length would be extended from initialized size if appending (and empty entries too)